docs/content/commands/npm-pkg.md

```diff
@@ -98,6 +98,13 @@ Returned values are always in **json** format.
     npm pkg set contributors[0].name='Foo' contributors[0].email='foo@bar.ca'
     ```
 
+    You may also append items to the end of an array using the special
+    empty bracket notation:
+
+    ```bash
+    npm pkg get contributors[].name='Bar' contributors[].email='bar@bar.ca'
```

Comment:
```suggestion
    npm pkg set contributors[].name='Bar' contributors[].email='bar@bar.ca'
```
i don't think this syntax would make much sense for `get` :-p

Reply:
ooops, good catch!